### PR TITLE
Cookiecutting - additional files

### PR DESCRIPTION
**Pull request by 8bitsam**
Added and removed various files to match the cookiecutter template. Notably removed things we're no longer using like conda-recipe/ and travis.yml

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Authors
+=======
+
```
Comment (sbillinge) [inline AUTHORS.rst:3]: I think we need to add the current statement about Billinge group authors here from cookie cutter (but don't delete other stuff.)

**Review by sbillinge — COMMENTED** [review]
please see inline comment.  Also, let's not remove the conda-forge stuff just yet, we may need some of it.  Move the files over rather than delete them from the git repo.

This PR is gettting too big again.  Try and keep like with like...just adding files....just deleting files from the db....modifying files.

**Comment by 8bitsam** [comment]
Do we want to split this into multiple commits then? Most of it was replacing the .txt files with the .rst counterparts, so nothing should be functionally different.

**Comment by sbillinge** [comment]
> Do we want to split this into multiple commits then? Most of it was replacing the .txt files with the .rst counterparts, so nothing should be functionally different.

yes, please do that, but also to remove reverts of large numbers of files etc..  Much cleaner and nicer.

**Comment by sbillinge** [comment]
btw @8bitsam it is ok to open multiple PRs at the same time, all smaller.

**Comment by 8bitsam** [comment]
Replaced with #43, #44, #45. Two of these have merge conflicts (due to file removal/moving).

**Pull request closed without merging**